### ISSUE TITLE
Saved filters no longer working in heresphere

### ISSUE THREAD

**Issue by GoldstarZV** [o-fl0w/stash-vr#17]
Saved filters aren't showing up in heresphere anymore now, I believe it's because of the [saved filter refactor](https://github.com/stashapp/stash/pull/4054) that was merged to develop recently

**Comment by slabking** [comment]
Seeing this, too. Looks like this was released with [0.23.0](https://github.com/stashapp/stash/releases/tag/v0.23.0).

**Comment by Kruk2** [comment]
@GoldstarZV @nerfdaderp @slabking 
try my fork, prebuild exe only for windows:
https://github.com/Kruk2/stash-vr/releases/tag/0.7.0

https://github.com/Kruk2/stash-vr
Check beginning of readme as my version is slightly customized to my needs (passthrough support, no VR tags, setup filters by name instead of id)

**Comment by Kruk2** [comment]
@nerfdaderp I didn't remove any dockerfiles so you should be able to build it yourself:
git clone https://github.com/Kruk2/stash-vr
export BUILD_VERSION=0.7.0
docker build -t Kruk2/stash-vr

and if you're using docker-compose, fix the image version in this line
https://github.com/Kruk2/stash-vr/blob/master/docker-compose.yml#L4C12-L4C33

**Comment by nerfdaderp** [comment]
> @nerfdaderp I didn't remove any dockerfiles so you should be able to build it yourself: git clone https://github.com/Kruk2/stash-vr && cd stash-vr export BUILD_VERSION=0.7.0 docker build -t Kruk2/stash-vr .
> 
> and if you're using docker-compose, fix the image version in this line https://github.com/Kruk2/stash-vr/blob/master/docker-compose.yml#L4C12-L4C33 should be: Kruk2/stash-vr:latest

That sadly did not work; I was getting a docker file error. I was able to fork your repo and then build off of that into docker hub, but oddly enough after deploying the new image, I still don't have filters. I'm just gonna give up on it I think. 

Error this time is  (13:25:46[0m [31mWRN[0m Skipped filter [36merror=[0m[31m\"FindFiltersById: FindSavedFilter: Filter not found\"[0m [36mfilterId=[0m21 [36mmod=[0mheresphere [36mop=[0mbg [36mroute=[0mindex )

I have tweeticoats' other app deployed, and while it lacks the tighter integration of this one, I guess it will have to hold me over until this stuff is ported into stash as native functionality.

Thanks for all your help!

**Comment by Kruk2** [comment]
This error is a different from the original. Looks like stash-vr wasn't able to find filters.
Are you sure that you have provided FILTERS variable? Maybe stash did recreate filters in db migration process so the IDs are different?
I would do the following:
1. create sections.txt file
1. in every line put filter name that you want to have in stash-vr (case sensitive)
1. After line 25. here:
https://github.com/Kruk2/stash-vr/blob/master/Dockerfile#L25C1-L25C37
add `COPY sections.txt ./`
1. rebuild docker image.

**Comment by nerfdaderp** [comment]
All set! I had previously either removed filters from env variables or added them in by name (testing). Going back to the old numbered method has me back in business. Thanks again so much for all your help.

**Comment by Kruk2** [comment]
No problem!
For the named-version of filters in sections.txt you have to add that one line in DOCKERILE.
Otherwise it will be missing in the docker

**Comment by GoldstarZV** [comment]
hey @Kruk2 I tried using your exe version, and I got this: 

```
D:\stash>stash-vr-release.exe
panic: open web/template/index.html: The system cannot find the path specified.

goroutine 1 [running]:
html/template.Must(...)
        C:/Program Files/Go/src/html/template/template.go:368
stash-vr/internal/api/web.init()
        I:/Other/stash-vr/internal/api/web/web.go:15 +0x87
```

I did set the graphql url and api key

**Comment by Kruk2** [comment]
Hi @GoldstarZV , sorry I missed your message.
Try to download this:
https://github.com/Kruk2/stash-vr/archive/refs/tags/0.7.0.zip
unzip
place the exe inside the folder.

**Comment by grumplin** [comment]
HI trying to run this ony my collection. 
DEOVR gives the error unsupported feile type
Heresphere is stuck and displays "encode" instead of stream

btw. i dont use filenames as instruction on projection (probably heres the problem) and i dont use the VR folder

**Comment by BollywoodMaster** [comment]
@nerfdaderp, could you explain step by step how you got the solution build and thereafter deploy to your NAS.

**Comment by nerfdaderp** [comment]
> @nerfdaderp, could you explain step by step how you got the solution build and thereafter deploy to your NAS.

An admitted github newb, but I ended up forking the branch and then copying in the updated code Kruk2 kindly provided.

While I have no plans to maintain, and indeed it sounds like Stash will eventually get Heresphere\DeoVR native support, you can feel free to use my docker release. Just update the image path for your container and you should be good to go.  nerfdaderp/stash-vr:latest

**Comment by o-fl0w** [comment]
[v0.7.0](https://github.com/o-fl0w/stash-vr/releases/tag/v0.7.0) should fix this issue with support for stash v0.23
